### PR TITLE
Adding: IPTV extension

### PR DESCRIPTION
**Pull request by Aayush9029**
## Description

Raycast extension for [iptv-org](https://github.com/iptv-org/) which lets one browse through hundreds of IPTV channels

## Screencast

CleanShot 2022-04-13 at 23 01 22@2x
CleanShot 2022-04-13 at 23 00 57@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/iptv/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "description": "Search online IPTV channels",
+    "icon": "command-icon.png",
+    "author": "Aayush9029",
+    "categories": [
+        "Media"
+    ],
+    "license": "MIT",
+    "commands": [{
+        "name": "index",
+        "title": "Search IPTV channels",
```
Comment (fe9lix) [inline extensions/iptv/package.json:14]:
```suggestion
        "title": "Search IPTV Channels",
```

File: extensions/iptv/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        "description": "Search and browse and play hundreds of free IPTV  channels.",
```
Comment (fe9lix) [inline extensions/iptv/package.json:15]:
```suggestion
        "description": "Search and browse and play hundreds of free IPTV channels.",
```

**Review by fe9lix — CHANGES_REQUESTED** [review]
Hey @Aayush9029, thanks for your contribution. Some comments: 

- There's some empty-view flickering when opening the command. You can set the isLoading property on the list initially to true to avoid that.

- I would suggest to remove the success toast message since it's not needed and just distracting?

Also left some inline comments regarding naming.

**Comment by Aayush9029** [comment]
@fe9lix I've made the changes you've requested, lmk if there's anything else that needs improvement!

**Pull request merged**